app/views/admin/consumer_apps/index.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class="crayons-card p-3 mb-5 mt-2">
+  <%= link_to "New Consumer App", new_admin_consumer_app_path, class: "crayons-btn float-right" %>
+
+  <h1 class="crayons-title mb-6">Consumer Apps</h1>
+
+  <p>
+    The following are apps that will be able to receive Push Notifications from <%= SiteConfig.community_name %>.
+  </p>
+</div>
+
+<table class="crayons-table" width="100%">
+  <thead>
+    <tr>
+      <th scope="col">App Bundle</th>
+      <th scope="col">Platform</th>
+      <th scope="col">Devices</th>
+      <th scope="col">Authentication Key</th>
+      <th scope="col">Operational</th>
+      <th scope="col">Actions</th>
+    </tr>
+  </thead>
+  <tbody class="crayons-card">
+    <% @apps.each do |app| %>
+        <tr>
+          <td><%= app.app_bundle %></td>
+          <td><%= app.platform %></td>
+          <td><%= app.devices.count %></td>
+          <% if app.auth_credentials.present? %>
+            <td><span class="crayons-icon" role="img" aria-label="Authentication Key is available">✅</span></td>
+          <% else %>
+            <td><span class="crayons-icon" role="img" aria-label="Authentication Key is available">❌</span></td>
```

Comment:
should this be "not available" ?

Reply:
For some reason I had in my mind it would read "is available `checkmark emoji`", but I think you're right! Thanks for the catch 🙌

Reply:
Ah ok! [aria-label](https://www.aditus.io/aria/aria-label/) tells the screen reader what's going on when the element itself is not descriptive